datapackage/cli_datapackage_validate.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+
```

Comment:
Other imports for Py2/3 consistency would be desirable: https://github.com/okfn/coding-standards#python-23